boxsdk/client/client.py

```diff
@@ -1666,3 +1666,68 @@ def create_metadata_template(self, display_name, fields, template_key=None, hidd
             session=self._session,
             response_object=response,
         )
+
+    @api_call
+    def create_zip(self, name, items):
+        """
+        Creates a zip of multiple files and folders.
+
+        :param name:
+            The name of the zip file to be created.
+        :type name:
+            `unicode`
+        :param items:
+            List of files or folders to be part of the created zip.
+        :type items:
+            `Iterable`
+        :returns:
+            A dictionary representing a created zip
+        :rtype:
+            :class:`dict`
+        """
+        # pylint: disable=protected-access
+        url = self._session.get_url('zip_downloads')
+        data_items = []
```

Comment:
```suggestion
        zip_file_items = []
```

Reply:
Between `data_items`, `data = {`, and `data=joson.dumps(data))` I thought renaming this variable would cut down on the usage of the word `data` here :^)